core/services/job/runner_integration_test.go

```diff
@@ -695,5 +695,19 @@ ds1 -> ds1_parse;
 		r, err = runner.ResultsForRun(context.Background(), runID)
 		require.NoError(t, err)
 		assert.Error(t, r[0].Error)
+
+		config.Config.Set("JOB_PIPELINE_MAX_TASK_DURATION", "10ns")
+		runnerTest := pipeline.NewRunner(pipelineORM, config)
+		jb = makeMinimalHTTPOracleSpec(t, cltest.NewEIP55Address().String(), cltest.DefaultPeerID, transmitterAddress.Hex(), cltest.DefaultOCRKeyBundleID, serv.URL, "")
+		jb.Name = null.NewString("a job 4", true)
+		err = jobORM.CreateJob(context.Background(), jb, jb.Pipeline)
+		require.NoError(t, err)
+		runID, err = runnerTest.CreateRun(context.Background(), jb.ID, nil)
+		require.NoError(t, err)
+		err = runnerTest.AwaitRun(context.Background(), runID)
+		require.NoError(t, err)
+		r, err = runnerTest.ResultsForRun(context.Background(), runID)
+		require.NoError(t, err)
+		assert.EqualError(t, r[0].Error, "error making http request: context canceled")
```

Comment:
This is not very helpful to node operators, can we pretty up this error message a bit?